reg_tests/test_files/ActLineSimple/ActLineSimple.norm.gold

```diff
@@ -1,10 +1,10 @@
-6.162977394373625e-06 1 0.125
-2.056149689290644e-06 2 0.25
-1.777391604459957e-06 3 0.375
-1.709792586888331e-06 4 0.5
-1.657172782293955e-06 5 0.625
-1.608096510227637e-06 6 0.75
-1.563632741945283e-06 7 0.875
-1.523637623960223e-06 8 1
-1.487544905975012e-06 9 1.125
-1.454680405759612e-06 10 1.25
+7.909774684098609e-16 1 0.125
```

Comment:
@psakievich Any idea why switching from elements to lines would cause such a large drop in the residuals?

Reply:
@jrood-nrel I'd just disable this test.  It is the same (minus the elem vs edge) as `actLineSimpleNGP` and I'm going to delete all the redundant legacy code when I address #837.